src/Components/SpotifyControl/SpotifyControl.js

```diff
@@ -0,0 +1,53 @@
+import {getPlaybackState, playSong, pauseSong} from "Services/Spotify/spotifyAPI";
+import {useState, useEffect} from "react";
+import useSpotifyAuth from "Store/selectors/useSpotifyAuthData"
+
+import SpotifyControlView from "./SpotifyControlView";
+
+const SpotifyControl = function () {
+  const token = useSpotifyAuth();
+
+  const [playBackState, setPlayBackState] = useState(null);
+  const [data, setData] = useState({
+    "image_src": process.env.PUBLIC_URL + "/bragi-icon.png",
+    "name": "Bragi",
+    "artist": "3000",
+    "is_playing": false,
+  });
+  const [isPlaying, setIsPlaying] = useState(false)
+
+  useEffect(() => {
+    if (token) {
+      setPlayBackState(getPlaybackState(token.access_token).then(handleData));
```

Comment:
When doing the thing recommended below, this should be updated to something like:

```js
getPlaybackState(token.access_token)
  .then((newData) => {
    if (newData.statusCode === 200)
      setPlaybackState(newData); // Or newData.body, as the rest is not really necessary
  });